fuzzbucket/__init__.py

```diff
@@ -84,6 +70,25 @@ def default(o: typing.Any) -> typing.Any:
         return DefaultJSONProvider.default(o)  # pragma: no cover
 
 
+@functools.cache
+def get_vpc_id(ec2_client) -> str:
+    value = cfg.vpc_id()
+
+    if value.startswith("vpc-"):
+        return value
+
+    candidate_vpcs = ec2_client.describe_vpcs(
+        Filters=[
+            dict(Name="tag:Name", Values=[value]),
+        ]
+    ).get("Vpcs", [])
+
+    if len(candidate_vpcs) == 0:
+        return value
```

Comment:
Is it a valid vpc id in this case?

Reply:
Hmmm yeah, probably not. Would it be better to use an explicitly bogus value like `"NOTSET"` and let the operation fail, or something else?

Reply:
Fail here versus fail in caller?
Return `None`?
Log a message here?